data/migrations/20190321213238_createQuantitiesTable.js

```diff
@@ -1,34 +1,24 @@
+const foreignId = (tbl, refTable, refColumn) => {
+  tbl
+    .integer(`${refColumn}_id`)
+    .notNullable()
+    .unsigned()
+    .references("id")
+    .inTable(refTable)
+    .onDelete("CASCADE")
+    .onUpdate("CASCADE");
+};
```

Comment:
You could move this foreignId helper into a separate file so that you can import it into your recipes migration as well (since that foreign id looks the same as well). For example, with some work still left up to you, in a data/migrationHelper.js file maybe:
```javascript
const someHelperFunc = () => { console.log('it works!') };

module.exports = {
  someHelperFunc,
};
```
and in the files where you want to use this helper function:
```javascript
const { someHelperFunc } = require('./path_to/migrationHelper');

// use it as you wish
someHelperFunc();
```